### PR TITLE
Logan/fix tests

### PR DESCRIPTION
**Pull request by logyball**
based on martin's changes to the directory structure, I've reworked how the unit and integration tests are structured so now they work.  I had to do a little bit of hacking together so that they run, and I'd like to do some more reworking to both eliminate IDE warnings (~~module not found because the directory structure is something i tell the tests about at runtime etc~~) and to make this more stable

~~also the behave tests all fail. but I'm working on resolving that.  At least they run...~~ jk they work

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,8 @@ env:
 #  - HEADLESS="true"
 
 install:
-  - pip install -r requirements.txt
+  - pip3 install pipenv
```
Comment (lixphilosophy) [inline .travis.yml]: While user has mutiple version of `python3`, `pip3` might point to a package manager than the one we want. For exmaple, as I test on one the lab machine:

```
$ python3 --version
Python 3.7.3

$ which python3
/home/newslab/anaconda3/bin/python3

$ pip3 --version
pip 19.1.1 from /usr/local/lib/python3.5/dist-packages/pip (python 3.5)

$ python3 -m pip --version
pip 19.0.3 from /home/newslab/anaconda3/lib/python3.7/site-package/pip (python 3.7)
```

A better approach would be `python3 -m pip ...`, if `pip3` would just be used at `line 12`

Comment (logyball) [reply .travis.yml]: good call I'll make that change

**Review by lixphilosophy — DISMISSED** [review]
All looks good, good jobs

**Review by logyball — COMMENTED** [review]
this still needs work to test out the different file structure things

**Comment by logyball** [comment]
I'm going to work on this more before this pr should be merged.  Somethings fishy here and I'd like to figure this out before submitting for review

**Pull request closed without merging**